plugins/hu.bme.mit.massif.communication.matlabengine/src/hu/bme/mit/massif/communication/matlabengine/MatlabEngineEvaluator.java

```diff
@@ -70,42 +70,56 @@ private IVisitableMatlabData convertToMatlabData(Object[] rawData) {
 			} else if (value instanceof String[]) {
 				CellMatlabData cellData = new CellMatlabData();
 				for (String stringVal : (String[]) value) {
-					cellData.addData(new MatlabString(stringVal));
+					cellData.addData(new MatlabString(stringVal.replaceAll("\n", " ")));
```

Comment:
Why do you remove newlines?

Reply:
Because MATLAB considers newlines and spaces the same. For us, it's easier to deal only with spaces.